include/odp/api/spec/packet.h

```diff
@@ -2140,6 +2140,50 @@ uint32_t odp_packet_payload_offset(odp_packet_t pkt);
  */
 int odp_packet_payload_offset_set(odp_packet_t pkt, uint32_t offset);
 
+/**
+ * Enable or disable Tx packet drop based on age
+ *
+ * Enable or disable Tx packet drop based on packet age. When enabled, packet will be dropped
+ * if it is in Tx pktout queue for longer than per-packet timeout.
+ *
+ * @param pkt     Packet handle
+ * @param enable  True indicates aging to be enabled on the packet.
+ */
+void odp_packet_aging_drop_set(odp_packet_t pkt, odp_bool_t enable);
```

Comment:
Is separate drop enable/disable calls needed (odp_packet_aging_drop_set() and odp_packet_has_aging_drop())? Zero nanoseconds is not a useful value for aging, so we could specify that nsec==0 means that aging is disabled.

Reply:
Ack, will make it to two API's(one for set and one for get) where 0 nsec means disable otherwise enable.